### ISSUE TITLE
Remove GNS3 VM specific code

### ISSUE THREAD

**Issue by grossmj** [GNS3/gns3-web-ui#1161]
The GNS3 server version 3.0 will not support the GNS3 VM endpoints and settings. We need to remove all the code related to this.

![Screenshot from 2021-06-03 18-30-52](https://user-images.githubusercontent.com/4752614/120618149-f090e480-c499-11eb-812d-92220835e7a8.png)

![Screenshot from 2021-06-03 18-36-27](https://user-images.githubusercontent.com/4752614/120619483-2bdfe300-c49b-11eb-9cb0-4694d295ff57.png)

**Comment by piotrpekala7** [comment]
should we apply any changes also for topology summary widget?

**Comment by grossmj** [comment]
Is there anything related to the GNS3 VM in the topology summary widget?

**Comment by piotrpekala7** [comment]
VM is displayed on servers tab

**Comment by piotrpekala7** [comment]
dialog to configure gns3vm also should be removed

**Comment by piotrpekala7** [comment]
![gns3vm](https://user-images.githubusercontent.com/31202938/121520705-81644480-c9f3-11eb-9e31-ca553c6d176d.PNG)
should we also remove first step from adding templates?

**Comment by piotrpekala7** [comment]
@grossmj ?

**Comment by grossmj** [comment]
Yes you can remove from adding templates but keep in mind that remote servers will have to be implemented at a later stage.

**Comment by piotrpekala7** [comment]
So maybe we should remove only option in first step?

**Comment by grossmj** [comment]
Yes it's fine 👍

**Comment by piotrpekala7** [comment]
will be fixed in https://github.com/GNS3/gns3-web-ui/pull/1168

**Comment by piotrpekala7** [comment]
https://github.com/GNS3/gns3-web-ui/pull/1168 merged into master-3.0

**Comment by grossmj** [comment]
Looks like there is still GNS3 VM code when I try to install a Docker appliance:

![Screenshot from 2021-08-11 17-44-15](https://user-images.githubusercontent.com/4752614/128994710-693bd7d5-fb15-4d2c-bc38-66945f6b3bd5.png)

![Screenshot from 2021-08-11 17-43-37](https://user-images.githubusercontent.com/4752614/128994720-f42f475c-0cca-4b87-83e5-bceb1eaee2d2.png)
